packages/button/src/LionButton.js

```diff
@@ -67,24 +52,35 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
     return [
       css`
         :host {
+          position: relative;
           display: inline-block;
           min-height: 40px; /* src = https://www.smashingmagazine.com/2012/02/finger-friendly-design-ideal-mobile-touchscreen-target-sizes/ */
-          outline: 0;
-          background-color: transparent;
           box-sizing: border-box;
+          vertical-align: middle;
+          line-height: 24px;
+          background: #eee; /* minimal styling to make it recognizable as btn */
+          padding: 8px; /* padding to fix with min-height */
+          outline: none; /* focus style handled below */
         }
 
-        .btn {
-          min-height: 24px;
+        :host::after {
```

Comment:
what does this do?

Reply:
Changed the the "click area".

About the `cursor: pointer`, it's use for links, the native button has default cursor. I'm not sure if we want to change the cursor for buttons 🤔

Reply:
So checked it in the browser and we only need:

:host::before {
  /* occupy full width of host */
  left: 0;
  right: 0;
}

.button-content {
    position: relative; /* we need to win in stacking order from other positioned element (::before), so we can still selecr text */
    cursor: default; /* we should always see the default arrow, never a caret */
}

Reply:
I added an example, taking care about min-height and min-width. I guess `translate` approach works better because create an area button centric, instead of top left corner

Reply:
Nice 💪  Makes sense